Makefile.am

```diff
@@ -3241,6 +3281,7 @@ intgcheck-prepare:
 	    --without-semanage \
 	    $(INTGCHECK_CONFIGURE_FLAGS); \
 	$(MAKE) $(AM_MAKEFLAGS); \
+	$(MAKE) $(AM_MAKEFLAGS) tcurl-test-tool || echo "libcurl tests will be disabled"; \
```

Comment:
What is an intention of this line?

I briefly checked the patch and curl seems to be optional build dependency for tests;
the same as cmocka.